server/views/pages/reportDetailMacro.njk

```diff
@@ -64,6 +64,16 @@
               dataValue: data.incidentDetails.incidentDate | formatDate('HH:mm') 
       })
     }}
+    {% if prison %}
```

Comment:
is this `if` required? I think we would always want to show the prison?

Reply:
Think we need to load the prison when displaying "Your Report" and "View Report" too

Reply:
done